### ISSUE TITLE
accessibility #17-#18 - patch facet.html.erb and _ajax_modal.html.erb, removing aria-hidden attributes enabling voiceovers

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#235]
#17 
The modal dialog that appears when the “more” link is activated under the filter heading “creator”:
(1)	The close (x) button on the modal dialog should be able to be activated via the Enter key or the Spacebar key. Elements should operate using the expected keystrokes.  This button element has an aria-hidden=”true” on it.  This hides the element from assistive technology, which hides its function as well.  A screen reader user would not be able to close this modal dialog.
(2)	Also, the close (x) button is not properly labeled.  The accessible name (if it wasn’t hidden) would be “x”.  An aria-label or screen reader only text should be added to give a name that describes its function (ie, “close”).

**Comment by yulgit1** [comment]
issue 18 

Description: | The modal dialog that appears when the “more” link is activated under the filter heading “creator”:The entire modal dialog has an aria-hidden=”true”.  This hides the dialog from assistive technology.  A screen reader user would hear nothing inside this dialog.

**Comment by yulgit1** [comment]
resolved: https://github.com/ycba-cia/blacklight-collections2/commit/b768966dd2ad5c2082295c3078c0afc92b37a9e0
